Backends/CLX-fb/frame-manager.lisp

```diff
@@ -37,10 +37,10 @@
         (setf concrete-pane-class (find-class class-symbol)))))
   concrete-pane-class)
 
-(defmethod make-pane-1 ((fm clx-fb-frame-manager) (frame application-frame) type &rest args)
-  (apply #'make-instance
-	 (maybe-mirroring fm (clim-clx::find-concrete-pane-class type))
-	 :frame frame
-	 :manager fm
-	 :port (port frame)
-	 args))
+(defmethod find-concrete-pane-class ((fm clx-fb-frame-manager)
+                                     pane-type &optional (errorp t))
+  ;; This backend doesn't have any specialized pane implementations
+  ;; but depending on circumstances it may add optional mirroring to
+  ;; the class. Such automatically defined concrete class has the same
+  ;; name but with a gensym prefix and symbol in the backend package.
+  (maybe-mirroring fm (find-concrete-pane-class t pane-type errorp)))
```

Comment:
Couldn't this `(call-next-method)` like the method for `clx-frame-manager`?

Reply:
no, because the next method is clx-frame-manager, which would be essentially:

`(clim-fb-clx::maybe-mirroring (clim-clx::maybe-mirroring (find-concrete-pane-class t pane-type errorp)))`

Reply:
Right.

Strangely though, `clim-fb-clx::maybe-mirroring` checks whether `concrete-pane-class` already is a subclass of `mirrored-sheet-mixin` as if it was intended to be idempotent. That might make `(clim-fb-clx::maybe-mirroring (clim-fb-clx::maybe-mirroring …))` work, but not `(clim-fb-clx::maybe-mirroring (clim-clx::maybe-mirroring …))`, so you are right.